docs/api-types/service-export.md

```diff
@@ -0,0 +1,38 @@
+# ServiceExport API Reference
+
+## Introduction
+
+In AWS Gateway API Controller, `ServiceExport` is the way of enabling a Service for multi-cluster traffic setup.
+Once a Service is exported, other clusters can refer to it using [`ServiceImport`](service-import.md) resource.
+
+ServiceExport is a term originated from Kubernetes [Multicluster Service APIs](https://multicluster.sigs.k8s.io/concepts/multicluster-services-api/);
+but AWS Gateway API Controller uses its own version of ServiceExport resource with a certain set of features.
+
+Internally, creating a ServiceExport creates a VPC Lattice [target group](https://docs.aws.amazon.com/vpc-lattice/latest/ug/target-groups.html).
+You can take advantage of this by using the created target group in VPC Lattice setup outside Kubernetes.
```

Comment:
> You can take advantage

That applies to everything that controller creates in Lattice. Probably be more specific what kind of advantage or just remove this sentence.

Reply:
I think it is important to know that creating a ServiceExport is an officially recommended method to create a target group. Will try to rephrase.